packages/gas-fee-controller/CHANGELOG.md

```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [18.0.0]
+
+### Added
+
+- **BREAKING:** Add constructor options to `GasFeeController`: `EIP1559APIEndpoint` (required), and `legacyAPIEndpoint` (optional) which defaults to `LEGACY_GAS_PRICES_API_URL`. ([#4446](https://github.com/MetaMask/core/pull/4446))
+  - These URLs are no longer hardcoded within the controller.
+
+### Removed
+
+- **BREAKING:** Remove `infuraAPIKey` as a constructor option for `GasFeeController`. This class field was previously used to construct and send the `Authorization` header for Infura gas API requests. ([#4446](https://github.com/MetaMask/core/pull/4446))
+- **BREAKING:** Remove all usage of the `providerConfig` state property of `NetworkController`. ([#4356](https://github.com/MetaMask/core/pull/4356))
+  - The `selectedNetworkClientId` state property of `NetworkController` can be used instead of `providerConfig` to derive the currently selected `chainId`.
```

Comment:
I believe we can omit this change. The changes here should have no consumer-facing consequences. (The gas fee controller does need to assume that the network controller has a `selectedNetworkClientId`, but that is a fair assumption based on the version of `network-controller` that `gas-fee-controller` depends on.)

``` suggestion
```